CustomApps/lyrics-plus/style.css

```diff
@@ -611,4 +612,4 @@ button.switch.small {
 }
 .lyrics-lyricsContainer-UnsyncedLyricsPage .split > div {
 	flex: 50%;
-}
+}
```

Comment:
> what happened here?
@ririxidev may have the fix for the workflow stuff, for now you can return new line at EOF here and it should be good